private/buf/cmd/buf/command/registry/organization/organizationcreate/organizationcreate.go

```diff
@@ -38,7 +38,7 @@ func NewCommand(
 ) *appcmd.Command {
 	flags := newFlags()
 	return &appcmd.Command{
-		Use:   name + " <buf.build/organization>",
+		Use:   name + " <remote/organization>",
```

Comment:
I think it might be better to have `<remote>/<organization>`, since they are two inputs separated by `/`, but this could be misinterpreted as one or the other.

Reply:
Actually, for consistency, let's keep it this way.